package.json

```diff
@@ -84,8 +84,6 @@
     "can-use-dom": "^0.1.0",
     "google-maps-infobox": "^1.1.13",
     "invariant": "^2.1.1",
-    "lodash": "^4.12.0",
-    "lodash.isequal": "^3.0.4",
```

Comment:
Oh. I knew I had to keep it. Strange it was removed because I double checked. Must have made a mistake. Going to fix it now.